scripts/setup.sh

```diff
@@ -2,21 +2,14 @@
 
 NODE_VERSION=v16.17.1
 
-echo "installing node ${NODE_VERSION}"
-if setup_service node $NODE_VERSION; then
-  echo "Installed node ${NODE_VERSION} successfully"
-else
-  echo "node ${NODE_VERSION} installation failed."
-fi
-
 # Note: Yarn will automatically switch over to yarn 3 after installing yarn 1.x
 YARN_VERSION=1.22.17
```

Comment:
Update to 1.22.19

Reply:
✅ 
f7d4bcb847005fa4641fff2ef3cffcf46b5e61a7